### PR TITLE
Intermediate Issue no. 134 Add content to CS/IT, electrical, mechanical, civil and electronics pages #134

### PR DESCRIPTION
**Pull request by fykaa**
Added Relevant Content to the 
1) CS/IT page
2) Electrical page
3) Electronics page
4) Mechanical page
5) Civil page

Fixed the broken links of these pages by addressing correct ids and buttons

Before- 
image
After-
screencapture-127-0-0-1-5500-src-pages-academics-electrical-html-2022-09-15-03_43_31

Daunting and took rigorous hours, works good now, pls accept pr :)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "livePreview.defaultPreviewPath": "c:\\Users\\Faeka\\Desktop\\Acadmica\\src\\pages\\academics\\csit.html"
```
Comment (narayan954) [inline .vscode/settings.json:2]: can you add this directory to the git ignore file ?

**Review by narayan954 — CHANGES_REQUESTED** [review]
you really did a lot of work! Kudos to you! can you do that last work as I mentioned ?

File: .vscode/settings.json (hunk 1 above)
Comment (fykaa) [reply .vscode/settings.json]: Hey, sure, I removed this since it wasn't supposed to be in the commits. Please have a look )

Comment (narayan954) [reply .vscode/settings.json]: yes that'd work!

**Review by narayan954 — APPROVED** [review]
looks great!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient

**Pull request merged**